Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -166,6 +166,25 @@ class OWTestLearners(widget.OWWidget):
     TARGET_AVERAGE = "(Average over classes)"
     class_selection = settings.ContextSetting(TARGET_AVERAGE)
 
+    class Error(OWWidget.Error):
+        class_required = Msg("Train data input requires a class variable")
+        class_required_test = Msg("Test data input requires a class variable")
+        too_many_folds = Msg("Number of folds exceeds the data size")
+        class_inconsistent = Msg("Test and train data sets "
+                                 "have different classes")
+
+    class Warning(OWWidget.Warning):
+        missing_data = Msg()
```

Comment:
Me neither. `Msg({})` doesn't look great, either. I'll define another class whose name will indicate that the whole message will be formed when it's raised, that is,

    missing_data = CustomMsg()